bookshelf_manager.py

```diff
@@ -0,0 +1,32 @@
+from bookshelf import Book
+from bookshelf import BookShop
+
+
+def main():
+    book1 = Book("Song of Ice and Fire", "J. Martin", 400, 200, 1000, 250)
+    book2 = Book("Harry Potter", "Joanne Rowling", 400, 100, 8000, 4000)
+    book3 = Book("JoJo Bizarre Adventures", "H. Araki", 900, 300, 10000, 8000)
```

Comment:
name of instances should be more understandable
like name of the book : Song_of_Ice_and_Fire = Book("Song of Ice and Fire", "J. Martin", 400, 200, 1000, 250)

Reply:
Renamed them👌